EWS should not be triggered for multi-year old patches

aj062 opened the pull request:
#### 6e27809f8901d7e0ade1b10baec354ff8444e309
<pre>
EWS should not be triggered for multi-year old patches
<a href="https://bugs.webkit.org/show_bug.cgi?id=243271">https://bugs.webkit.org/show_bug.cgi?id=243271</a>

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-app/ews/views/submittoews.py:
(SubmitToEWS.post):

Canonical link: <a href="https://commits.webkit.org/252916@main">https://commits.webkit.org/252916@main</a>
</pre>

aj062 commented on Tools/CISupport/ews-app/ews/views/submittoews.py:
```diff
@@ -45,11 +45,17 @@ def get(self, request):
             return HttpResponse('Invalid patch id provided, should be an integer. git hashes are not supported.')
 
-        _log.info('SubmitToEWS::change: {}'.format(change_id))
+        if change_id < 430000:
```
430000 is around May-2021. We can adjust it if anyone feel it's not very appropriate.

youssefsoli replied:
If it's possible to fetch the latest patch date, that would be more ideal since this blocks old bugs from being picked up again and tested within Bugzilla. (but since we want to deprecate this anyways, I am not sure)

youssefsoli replied:
and by picked up I mean re-submitted as a patch to the same Bugzilla.

aj062 replied:
It's possible but Alexey had another good point about ChangeLog. Went with that.

ryanhaddad commented on Tools/CISupport/ews-app/ews/views/submittoews.py:
```diff
@@ -45,11 +45,17 @@ def get(self, request):
+        if change_id < 459500:
+            return HttpResponse('Patch is too old. Skipping.')
+
+        if change_id > 600000:
```
Setting a time limit on deprecating patch based workflows, eh?

aj062 replied:
Yeah 😆 
Did very rough calculations. It's 10+ year in future based on current patch rate.
Just wanted to filter any garbage input while I was at it.

aj062 committed:
EWS should not be triggered for multi-year old patches
https://bugs.webkit.org/show_bug.cgi?id=243271

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-app/ews/views/submittoews.py:
(SubmitToEWS.post):

Canonical link: https://commits.webkit.org/252916@main
